ksp_plugin_test/plugin_integration_test.cpp

```diff
@@ -54,14 +54,15 @@ class PluginIntegrationTest : public testing::Test {
         looking_glass_(Permutation<ICRFJ2000Equator, AliceSun>::XZY),
         solar_system_(
             SolarSystemFactory::AtСпутник1Launch(
-                SolarSystemFactory::Accuracy::AllBodiesAndOblateness)),
+                SolarSystemFactory::Accuracy::MinorAndMajorBodies)),
         initial_time_(Instant() + 42 * Second),
         sun_body_(make_not_null_unique<MassiveBody>(
             MassiveBody::Parameters(solar_system_->gravitational_parameter(
                 SolarSystemFactory::name(SolarSystemFactory::Sun))))),
         planetarium_rotation_(1 * Radian),
         plugin_(make_not_null_unique<Plugin>(initial_time_,
                                              planetarium_rotation_)) {
+    // A subterranean satellite.
```

Comment:
Is it actually subterranean? it looks like the norm of that displacement is 7931.86 km